package.json

```diff
@@ -14,11 +14,13 @@
   },
   "main": "index.js",
   "scripts": {
+    "lint": "eslint src",
+    "fix": "eslint src --fix",
```

Comment:
Do dev need to run these before building? Can we automate these as part of build? And also I think we need to update Readme to add these